pkg/cmd/server/origin/controller.go

```diff
@@ -86,6 +86,11 @@ func (c *MasterConfig) NewKubernetesControllerInitalizers(kc *kubernetes.MasterC
 		ret[name] = initFn
 	}
 
+	// NB: this is funky -- it's actually a Kubernetes controller, but we run it as an OpenShift controller in order
+	// to get a handle on OpenShift clients, so that our delegating scales getter can work.  Ergo,
+	// we need to just not do anything right now.
+	delete(ret, "horizontalpodautoscaling")
```

Comment:
for example, we override the `serviceaccount` controller without deleting from this list